### DISCUSSION TITLE
[Mimir] Why out-of-the-box dashboards are set to UTC timezone?

### DISCUSSION THREAD

**Discussion by Rohlik** [grafana/mimir#5831] [Help and support] [answered]
Hello there 👋🏼,
I noticed that for some reason, included Grafana dashboards for Mimir contains explicitly UTC timezone, like [here](https://github.com/grafana/mimir/blob/ece52edeba00e3b797b94ad73dd4ce4574db812b/operations/mimir-mixin-compiled/dashboards/mimir-alertmanager-resources.json#L832). But why? 🤔
Why can we leave the default there and let the user decide which timezone she/he/it wants based on their global timezone settings in Grafana? 🤔

**Comment by pstibrany** [comment, marked as answer]
The reason is that we're operating many Mimir clusters in many locations around the world, and using single timezone for dashboards makes it easier to reason about.

However if there's an easy way to make this configurable in our Jsonnet (but still default to UTC), we would accept the patch.
